src/main/java/org/terasology/launcher/util/IBuildRepository.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.launcher.util;
+
+import org.terasology.launcher.game.GameJob;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.List;
+
+public interface IBuildRepository {
```

Comment:
I don't like prefixing interfaces with a capital `I` - the compiler knows this little details, and so does (or should) your IDE. From the usage site, you are working with a `BuildRepository` and you don't care whether it's an interface, an abstract class, or a rubber duck 🦆 
```suggestion
public interface BuildRepository {
```

Reply:
This should also have some Javadoc while we are at it 😉